awx/main/utils/receptor.py

```diff
@@ -37,6 +37,7 @@ def worker_info(node_name, work_type='ansible-runner'):
     kwargs = {}
     if work_type != 'local':
         kwargs['ttl'] = '20s'
+        kwargs['signwork'] = True
```

Comment:
you'll need signwork=True in local case as well. So you can just put signwork=True in the submit_work down below

Reply:
I have made the required changes